### PR TITLE
Remove mirna, motif, peak steps from BuildRegulationMart

### PR DESCRIPTION
**Pull request by luca-drf**

### CODE REVIEW

File: modules/Bio/EnsEMBL/PipeConfig/BuildRegulationMart_conf.pm [hunk 1]
```diff
@@ -127,14 +127,8 @@ sub pipeline_analyses {
...
-        1 => WHEN(
-          '(#dataset# ne "dmelanogaster")' => ['AddExtraMartIndexesExternalFeatures',
-                                               'AddExtraMartIndexesPeaks',
-                                               'AddExtraMartIndexesMiRNATargetFeatures',
-                                               # 'AddExtraMartIndexesMotifFeatures',
-                                               'AddExtraMartIndexesRegulatoryFeatures'],
-          ELSE 'AddExtraMartIndexesExternalFeatures',
-        )
+        1 => ['AddExtraMartIndexesExternalFeatures',
+              'AddExtraMartIndexesRegulatoryFeatures'],
```
Comment (marcoooo) [inline modules/Bio/EnsEMBL/PipeConfig/BuildRegulationMart_conf.pm:131]: Not sure about the peaks though. I think I was obliged to reintroduce the creation of this datasets for 104.

**Review by marcoooo — COMMENTED** [review]
In anycase, I believe this PR should be opened against master? Or is there another one coming in from 104 --> master?

**Comment by luca-drf** [comment]
> In anycase, I believe this PR should be opened against master? Or is there another one coming in from 104 --> master?

Since https://github.com/Ensembl/ensembl-biomart/pull/140 and https://github.com/Ensembl/ensembl-biomart/pull/139 were merged onto `release/104` I thought that was the case for this one as well. Should I open one for `master`?

File: modules/Bio/EnsEMBL/PipeConfig/BuildRegulationMart_conf.pm (hunk 1 above)
Comment (luca-drf) [reply modules/Bio/EnsEMBL/PipeConfig/BuildRegulationMart_conf.pm:131]: ENSPROD-6705 states that mirna, peaks and motif features aren't needed anymore. Should I put peaks back?

**Pull request closed without merging**